### ISSUE TITLE
Replace  a Gaten Error Property

### ISSUE THREAD

**Issue by Lmisel** [Uuvana-Studios/longvinter-windows-client#1040]
Replace Gaten
Quit a Gaten to replace for Gaten Security.

**To Reproduce**
1. Go to 'your own Gaten'
2. Click on 'Gaten to quit'
3. Click    'to put New Gaten in your property'
4. See an error 

**Expected behavior**
If i wanna upgrade my house and wanted changed to be more secure  cant do it because this error compromise the changes.

**Screenshots**
![image](https://user-images.githubusercontent.com/72278213/178022425-83181f7a-2a38-477b-9eb9-5a4fdbf4718f.png)

**Desktop (please complete the following information):**
 - OS: [Windows 10 Pro]
 - Game Version [Beta. 1.0.8]

**Comment by faketucker** [comment]
Can you try to relog and place it again?

**Comment by Lmisel** [comment]
Yes, well finally try to  refresh my host server, i think  maybe the error become when other peoples join to own server  while doing changes in your property, i try to do it again and try to report the feedback
